via/services/google_drive.py

```diff
@@ -0,0 +1,123 @@
+import json
+import re
+from io import BytesIO
+from json import JSONDecodeError
+from typing import ByteString, Iterator
+
+from google.oauth2.service_account import Credentials
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+from googleapiclient.http import MediaIoBaseDownload
+from pyramid.httpexceptions import HTTPNotFound
+
+from via.exceptions import ConfigurationError, UpstreamServiceError
+
+
+class GoogleDriveAPI:
+    """Simplified interface for interacting with Google Drive."""
+
+    SCOPES = [
+        # If we want metadata
+        "https://www.googleapis.com/auth/drive.metadata.readonly",
+        # To actually get the file
+        "https://www.googleapis.com/auth/drive.readonly",
+    ]
+
+    def __init__(self, service_account_info):
```

Comment:
This is where the name "Google drive credentials" changes to "service account info" for the same object. I think it'd be easier to follow if we stuck to one name